src/tools/kata-ctl/src/main.rs

```diff
@@ -35,12 +57,19 @@ fn real_main() -> Result<()> {
         Commands::Iptables(args) => handle_iptables(args),
         Commands::Metrics(args) => handle_metrics(args),
         Commands::Version => handle_version(),
+    };
+
+    // Log errors here, then let the logger go out of scope in main() to ensure
+    // the asynchronous drain flushes all messages before exit()
+    if let Err(ref e) = res {
```

Comment:
why `ref e`? `error!()` could take a reference.

Reply:
Fixed to `Err(e) = &res`, thanks.